.buildkite/release-pipelines/complete-code-freeze.yml

```diff
@@ -11,11 +11,15 @@ steps:
   - label: Complete Code Freeze
     plugins: [$CI_TOOLKIT]
     command: |
-      .buildkite/commands/configure-environment.sh
+      echo '--- :robot_face: Use bot for git operations'
+      source use-bot-for-git
 
       echo '--- :git: Checkout Release Branch'
       .buildkite/commands/checkout-release-branch.sh "$RELEASE_VERSION"
 
+      echo '--- :ruby: Setup Ruby Tools'
+      install_gems
```

Comment:
By the way, I think this is the best approach to run `bundle install`. That is, we should always run it _after_ checking out the release branch. Otherwise, we might run into the edge case of having the branch that triggered the build using a gemset different from the release branch which could lead to gnarly issues.

Reply:
🎯 

This is something I noticed in a lot of pipelines across our repos, that `install_gems` was running before checking out the new branch, so I've changed it with the `use-bot-for-git` change.